DenunciaCiudadana/src/navigation/IsLogin.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { View, Text} from 'react-native'
+import HomeScreen from '../screens/HomeScreen';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { NavigationContainer } from '@react-navigation/native';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+
+const Tab = createBottomTabNavigator();
+
+//Provisional screen
+function ListComplaintScreen() {
+    return (
+      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+        <Text>My list of complaints</Text>
+      </View>
+    );
+  }
+//provisional screen
+  function AddComplaintScreen() {
+    return (
+      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+        <Text>Add New complaint!</Text>
+      </View>
+    );
+  }
+
+  function SettingsScreen() {
+    return (
+      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+        <Text>Settings!</Text>
+      </View>
+    );
+  }
+
+const IsLogin = () => {
+    return (
+    <NavigationContainer>
+        <Tab.Navigator
+            screenOptions={({ route }) => ({
+              tabBarIcon: ({ focused, color, size }) => {
+                let iconName;
+                if (route.name === 'Home') {

```

Comment:
here happen something similar, we could improve the readability using a `switch` statement, maybe in another iteration, we could improve the `switch` by a `hash`.

Reply:
Ready I make a helper thath help me to dont have many code in the same component